openhcl/underhill_attestation/src/lib.rs

```diff
@@ -50,80 +51,80 @@ use zerocopy::FromZeroes;
 /// An attestation error.
 #[derive(Debug, Error)]
 #[error(transparent)]
-pub struct Error(ErrorInner);
+pub struct Error(UnderhillAttestationErrorInner);
 
-impl<T: Into<ErrorInner>> From<T> for Error {
+impl<T: Into<UnderhillAttestationErrorInner>> From<T> for Error {
     fn from(value: T) -> Self {
         Self(value.into())
     }
 }
 
 #[derive(Debug, Error)]
-enum ErrorInner {
-    #[error("read security profile from vmgs")]
+enum UnderhillAttestationErrorInner {
```

Comment:
meta: the term Underhill has been deprecated, and we should be working to migrate the codebase over to using OpenHCL terminology whenever possible. The only exceptions are in cases where we are tweaking / adding a new type in the context of code that is already predominantly using Underhill terminology, and we want to preserve consistency in that particular file / module / crate (until such a time that someone does a follow-up pass to do the renames).

This is a interesting case, because the crate name is still `underhill_`, but I do think its OK for the types inside a crate to get switched over, even if the crate itself is not yet renamed.

* * *

That said - as per my other comment, I don't believe this sort of type rename is even reasonable here.